src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -581,6 +601,26 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
                     returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
                     targetFunction: Guid_UO)
             },
+            {
+                BuiltinFunctionsCore.HEX2DEC,
+                StandardErrorHandling<StringValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: ReplaceBlankWithZero,
```

Comment:
>ReplaceBlankWithZero [](http://example.com/codeflow?start=40&length=20)

Hex2Dec takes a string as an input, so we should not replace blanks with a number